fastestimator/network/loss.py

```diff
@@ -19,13 +19,12 @@ class Loss:
     def __init__(self):
         pass
 
-    def calculate_loss(self, batch, prediction, mode):
+    def calculate_loss(self, batch, state):
         """this is the function that calculates the loss given the batch data
         
         Args:
             batch (dict): batch data before forward operation
```

Comment:
this is now batch data after forward op right?

Reply:
yep

Reply:
(comment was to point out the doc string is wrong)